tests/Issues/AutoImport/Fixture/skip_short_open_tag.php.inc

```diff
@@ -1 +0,0 @@
-<?= ns\Class::method() ?>
```

Comment:
why the fixture removed? It is valid code, see issue and PR:

- https://github.com/rectorphp/rector/issues/8425
- https://github.com/rectorphp/rector-src/pull/5496

Reply:
See https://github.com/rectorphp/rector-src/pull/6049/commits/d96bff4a3f570af8fcc80c9e14ec8fa3838161f6

and https://github.com/rectorphp/rector-src/pull/6049/commits/f08eff98157750fcfafd58776d6d4d5e6a7af61f

Reply:
No need to check every commit. I'll ping you when this is ready :)